server/store/templateHelperFuncs.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package store
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mattermost/mattermost-plugin-user-survey/server/model"
+)
+
+func (s *SQLStore) genAddColumnIfNeeded(tableName, columnName, dataType, constraint string) (string, error) {
+	tableName = addPrefixIfNeeded(tableName, s.tablePrefix)
+	normalizedTableName := s.normalizeTablename(tableName)
+
+	switch s.dbType {
+	case model.DBTypeMySQL:
```

Comment:
@harshilsharma63 Why are we adding MySQL support if we're deprecating it? Same question in the `genDropColumnIfNeeded` function

Reply:
Unfortunately some customers are still on MySQL and we want them to use this plugin.